src/test/java/uk/gov/justice/hmpps/offenderevents/e2e/HMPPSDomainEventsTest.java

```diff
@@ -152,7 +152,7 @@ void willPublishHMPPSDomainEvent() {
         class WhenIsReportedAsSentenced {
             @BeforeEach
             void setUp() {
-                PrisonApiExtension.server.stubPrisonerDetails("A5194DY", "SENTENCED", false, "ADM", "N", "ACTIVE IN", "MDI");
+                PrisonApiExtension.server.stubPrisonerDetails("A5194DY", "SENTENCED", false, "ADM", "K", "ACTIVE IN", "MDI");
```

Comment:
by bad luck I chose "N" (remand)  as the default movement code all over the place, so this is now replaced by a movement code we don't care about ("K") in a few tests.